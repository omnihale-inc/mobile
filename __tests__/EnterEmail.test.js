import renderer from "react-test-renderer";

import EnterEmail from "../app/create/index";

it("render enter email screen", () => {
  const tree = renderer.create(<EnterEmail />).toJSON();
  expect(tree).toMatchSnapshot();
});
