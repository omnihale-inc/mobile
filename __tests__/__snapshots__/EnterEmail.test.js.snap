// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render enter email screen 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "space-between",
      "paddingTop": 66.7,
    }
  }
>
  <View
    style={
      [
        {
          "backgroundColor": "#fff",
        },
        {
          "marginHorizontal": 24,
        },
      ]
    }
  >
    <Text
      style={
        [
          {
            "color": "#000",
          },
          {
            "fontFamily": "SatoshiBold",
            "fontSize": 20,
          },
        ]
      }
    >
      Enter your email Address
    </Text>
    <Text
      style={
        [
          {
            "color": "#000",
          },
          {
            "fontFamily": "SatoshiRegular",
            "fontSize": 14,
            "marginTop": 10,
          },
        ]
      }
    >
      A 4-code OTP will be sent to your email to verify your account.
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": "#fff",
          },
          {
            "position": "relative",
          },
        ]
      }
    >
      <TextInput
        keyboardType="email-address"
        onChangeText={[Function]}
        placeholder="Enter Email address"
        style={
          {
            "borderColor": "#d1d3d9",
            "borderRadius": 30,
            "borderWidth": 1,
            "fontFamily": "SatoshiRegular",
            "fontSize": 17,
            "marginVertical": 24,
            "paddingLeft": 55,
            "paddingRight": 40,
            "paddingVertical": 15,
          }
        }
        value=""
      />
      <View
        style={
          [
            {
              "backgroundColor": "#fff",
            },
            {
              "left": 22,
              "position": "absolute",
              "top": 43,
            },
          ]
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
        >
          <ViewManagerAdapter_ExpoImage
            contentFit="cover"
            contentPosition={
              {
                "left": "50%",
                "top": "50%",
              }
            }
            height={22}
            onError={[Function]}
            onLoad={[Function]}
            onLoadStart={[Function]}
            onProgress={[Function]}
            placeholder={[]}
            source={[]}
            style={
              {
                "height": 22,
                "width": 22,
              }
            }
            transition={null}
            width={22}
          />
        </View>
      </View>
    </View>
    <View
      style={
        [
          {
            "backgroundColor": "#fff",
          },
          {
            "flex": 0,
            "flexDirection": "row",
            "justifyContent": "space-between",
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#000",
            },
            {
              "fontFamily": "SatoshiRegular",
            },
          ]
        }
      >
        Have an account with us?
      </Text>
      <Text
        style={
          [
            {
              "color": "#000",
            },
            {
              "color": "#237e5d",
              "fontFamily": "SatoshiMedium",
            },
          ]
        }
      >
        Log in
      </Text>
    </View>
  </View>
  <View
    style={
      [
        {
          "backgroundColor": "#fff",
        },
        undefined,
      ]
    }
  >
    <Text
      style={
        [
          {
            "color": "#000",
          },
          {
            "backgroundColor": "#d1d3d9",
            "borderRadius": 30,
            "color": "#000000",
            "fontFamily": "SatoshiMedium",
            "fontSize": 14,
            "marginBottom": 40,
            "marginHorizontal": 24,
            "paddingVertical": 22,
            "textAlign": "center",
          },
        ]
      }
    >
      Send code
    </Text>
  </View>
</RCTSafeAreaView>
`;
